src/util/artifacts.js

```diff
@@ -1,14 +1,19 @@
 /* @flow */
-import {fs} from 'mz';
+import {fs as defaultFS} from 'mz';
```

Comment:
We currently use `mz/fs` everywhere else simply as `fs`. I would like to keep it consistent.